src/TLuaInterpreter.h

```diff
@@ -541,6 +541,9 @@ class TLuaInterpreter : public QThread
     static int spellSuggestWord(lua_State*);
     static int getDictionaryWordList(lua_State*);
     static int getTextFormat(lua_State*);
+    static int getPlayerName(lua_State*);
```

Comment:
Well, we already have `getPlayerRoom()` and there are uses of "player" in the translations in `setupLanguageData` though I am not sure if they are user visible. The tooltip (at least in the English version) for set location on the 2D map is "Set player current location to here"  Besides, "character" is three letters longer... the only place it is used is on the connect profiles dialogue and as the tooltips for name and password both need fixing because they lack possessive apostrophes it would not be amiss to change them:

For `Character name:` field from:
`The characters name` - which is stupidly redundent
to:
`The identifier for this profile's player in the game; if given, it can be supplied on connection to enable automatic log-in to the game together with a password entered in the next field.`

For `Password:` field from (in the portable storage mode):
`Characters password. Note that the password isn't encrypted in storage`
To:
`The secret text needed to log into the game in order to play this profile's player; if given, it can be supplied on connection to enable automatic log-in to the game.`
` `
`For more advanced situations it is now possible to record it here and allow it to be sent under restricted circumstances, via a command in a script. For security it is not possible for the script or any installed package or module to find out the password is. Further details are available in the Wiki.`
` `
`Note that the password isn't encrypted in storage.`

Reply:
Vadi asked me which one I'd prefer, so here I am. I'd prefer `character` as Vadim said, the player is the physical person playing the game, while the imagined person they control is a character (or avatar, but that is much less common and a third variant) in my book.

Reply:
concer, `character` is the more correct term, and since we haven't been entirely consistent for either one we can pick the proper one to use moving forward.

Reply:
Not a fan to blow up the tooltips to three paragraphs here, and not even mentioning the search term for the wiki

Reply:
I felt that three paragraphs (one of which is determined by the chosen password storage system) was the most readable I could come up with, but I could run the other two together:

`The secret text needed to log into the game in order to play this profile's player; if given, it will be supplied on connection to enable automatic log-in to the game. For more advanced situations it can be sent via a function` 'send~~Player~~CharacterPasssword()' `in a script. For security it is not possible for the script or any installed package or module to find out what the password is. Further details are available in the Wiki.`
` `
`Note that the password isn't encrypted in storage.`

However that looks worse to my mind than:

`The secret text needed to log into the game in order to play this profile's player; if given, it will be supplied on connection to enable automatic log-in to the game.`
` `
`For more advanced situations it can be sent via a function` 'send~~Player~~CharacterPasssword()' `in a script. For security it is not possible for the script or any installed package or module to find out what the password is. Further details are available in the Wiki.`
` `
`Note that the password isn't encrypted in storage.`

So we will change to `getCharacterRoom()` when we finally clean up the Lua API completely in a backward breaking way for version 2.0 in Mudlet 6.0 (when we also switch to Lua 5.3 or 5.4)? :grinning:

Reply:
Using the above and changing "can" for "will" for the name in the next commit:
`The identifier for this profile's player in the game; if given, it will be supplied on connection to enable automatic log-in to the game together with a password entered in the next field.`

Reply:
I meant the amount of words, not the presence of a new line :)